Overview.bs

```diff
@@ -1081,36 +1085,40 @@ in an extended font subset:
     space that covers at least the union of the axis space the client has and the axis space the
     client needs.</span>
 
-Additionally:
-
-*  <span class="conform server" id="conform-response-patch-format">The format of the patch in the
-    either the <code>patch</code> or <code>replace</code> fields must be one of those listed in
-    <code>accept_patch_format</code></span>.
+If the checksum of the original font computed by the procedure in [[#computing-checksums]] does not
+match the checksum in <code>PatchRequest.original_font_checksum</code> or
+<code>PatchRequest.original_font_checksum</code> is unset, then:
 
 *  <span class="conform server" id="conform-response-original-checksum">The value of
-    <code>original_font_checksum</code> must be set to the checksum of the original font.
-    The checksum value must computed by the procedure in [[#computing-checksums]].</span>
-
-*  <span class="conform server" id="conform-response-patched-checksum">
-     If <code>patch</code> or <code>replacement</code> fields are set the value of
-     <code>patched_checksum</code> must be set to the checksum of the extended font subset.
-     The checksum value must computed by the procedure in [[#computing-checksums]].</span>
+    <code>original_font_checksum</code> must be set to the checksum of the original font
+    computed by the procedure in [[#computing-checksums]].</span>
 
 *  <span class="conform server" id="conform-response-codepoint-ordering">
-    If the set of codepoints the client has is empty the response must set the
+    The response must set the
     <code>codepoint_ordering</code> and <code>ordering_checksum</code> fields following
     [[#codepoint-reordering]].</span>
 
 *  <span class="conform server" id="conform-response-original-features">
-    If the <code>patch</code> or <code>replacement</code> fields are set then the
-    <code>original_features</code> field should be set to the list of feature tags that the original
-    font has data for.</span>
+    The response must set the <code>original_features</code> field to the list of
+    <a href="https://docs.microsoft.com/en-us/typography/opentype/spec/featuretags">opentype layout feature tags</a>
+    that the original font has data for.</span>
```

Comment:
what if the original font and the new, replacement font have different feature sets?

Reply:
That's the motivation for this change. So how this works after this change is:
1. Say client has state for v1 of the font including original_feature list (v1).
2. At some point the server updates to v2 of the font which has a different feature list.
3. Client sends a patch request, server sees a mismatch between original checksum from the client and v2 of the font.
4. Server responds with  a patch that upgrades the clients v1 subset to the v2 subset with all features the client has previously asked for. This response will also include the original_features (v2) list (since it's mandatory to send when the checksum changes).
5. When client sees original_feature (v2) list included in the response it will replace the one previously saved list from v1 in its state.
6. Lastly, the client could check if there's features that it wants which are newly available in the v2 list and could potentially send an additional request to add them.